x-pack/plugins/enterprise_search/public/applications/workplace_search/components/shared/assets/source_icons_full_bleed/index.ts

```diff
@@ -0,0 +1,5 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
```

Comment:
Is this file needed?

Reply:
It's not. I had some weird TypeScript issues that were due to changes on master and was troubleshooting and this got left in. WIll remove. Thanks!